multi/stm32l4-multi/rng.c

```diff
@@ -0,0 +1,163 @@
+/*
+ * Phoenix-RTOS
+ *
+ * STM32L4 RNG driver
+ *
+ * Copyright 2024 Phoenix Systems
+ * Author: Aleksander Kaminski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <board_config.h>
+#include <stdint.h>
+#include <errno.h>
+#include <string.h>
+#include <sys/threads.h>
+#include <sys/platform.h>
+#include <sys/interrupt.h>
```

Comment:
Redundant include `sys/interrupt.h`, `interrupt()` function not used in `rng.c` module.
```suggestion
```